spring-cloud-huawei-config/src/main/java/com/huaweicloud/config/ConfigWatch.java

```diff
@@ -48,19 +48,19 @@ public void setApplicationEventPublisher(ApplicationEventPublisher applicationEv
 
   @Subscribe
   public void onConfigurationChangedEvent(ConfigurationChangedEvent event) {
-    LOGGER.info("receive new configurations, added=[{}], updated=[{}], deleted=[{}]",
-        event.getAdded().keySet(),
-        event.getUpdated().keySet(),
-        event.getDeleted().keySet());
-
+    Set<String> addKeys = event.getAdded().keySet();
+    Set<String> updateKeys = event.getUpdated().keySet();
+    Set<String> deleteKeys = event.getDeleted().keySet();
+    if (addKeys.isEmpty() && updateKeys.isEmpty() && deleteKeys.isEmpty()) {
+      return;
+    }
+    LOGGER.info("receive new configurations, added=[{}], updated=[{}], deleted=[{}]", addKeys, updateKeys, deleteKeys);
     Set<String> updatedKey = new HashSet<>();
     updatedKey.addAll(event.getAdded().keySet());
     updatedKey.addAll(event.getUpdated().keySet());
     updatedKey.addAll(event.getDeleted().keySet());
     ConfigRefreshEvent configRefreshEvent = new ConfigRefreshEvent(this, updatedKey);
     // publish RefreshEvent first to make sure spring bean refreshed
     applicationEventPublisher.publishEvent(new RefreshEvent(this, configRefreshEvent, "Config refreshed"));
-    // then invoke service listeners
```

Comment:
Can not remove this statement. For more information, see why higher versions made this change: https://github.com/huaweicloud/spring-cloud-huawei/commit/8e640eeb2ec7acf8552a884f5db486ab97540ded